marathon/app.go

```diff
@@ -96,6 +98,31 @@ type Task struct {
 	HealthCheckResults []HealthCheckResult `json:"healthCheckResults"`
 }
 
+func (t Task) GetMetric(prefix string) string {
+	taskAppID := string(t.AppID)
+
+	var appID string
+	if prefix == "" {
+		appID = taskAppID
+	} else {
+		appID = strings.Replace(taskAppID, prefix, "", 1)
+	}
+	noRootAppID := dropRootSlashID(appID)
+	metricPath := strings.Replace(noRootAppID, metrics.PathSeparator, metrics.MetricSeparator, -1)
+	taskStatus := strings.ToLower(t.TaskStatus)
+
+	return strings.Join([]string{metricPath, taskStatus}, metrics.MetricSeparator)
+
+}
+
+// dropRootSlashDI drops a beginning '/' from appID
+func dropRootSlashID(data string) string {
```

Comment:
I'd prefer [`strings.TrimPrefix`](https://golang.org/pkg/strings/#TrimPrefix) instead of this.